test_opensearchpy/test_async/test_server/test_rest_api_spec.py

```diff
@@ -54,6 +54,10 @@
 
 
 async def await_if_coro(x: Any) -> Any:
+    """
+    awaits if x is a coroutine
```

Comment:
Do we want to be consistent with full sentences? e.g. "Awaits if x is a coroutine."

Reply:
i want to get rid of the comment altogether and rename the function to "await_if_param_is_coroutine"